draft-ietf-teep-architecture.md

```diff
@@ -100,7 +100,7 @@ TEE. Typically, application components are chosen to execute inside a TEE becaus
 those application components perform security sensitive operations or operate on
 sensitive data. An application component running inside a TEE is referred to as a
 Trusted Application (TA), while an application running outside any TEE
-is referred to as an Untrusted Application.
+is referred to as an Untrusted Application (UTA).
```

Comment:
I do not think we should introduce an acronym here.  (We already got rid of "UA" as an acronym in earlier PRs.)